src/screens/Login/Start/components/HeaderLanguageSelectStyles.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import { StyleSheet } from 'react-native'
+// import colors from '../../common/colors'
```

Comment:
Should comments be removed?

Reply:
I suggest to keep this import commented in each *Styles.js file (if not used of course).
Right now we do not use colors, but it will be used soon.